docs/pages/installation/eks.md

```diff
@@ -6,15 +6,15 @@ permalink: installation/eks
 
 # Quick Start with Amazon Elastic Kubernetes Service (EKS)
 
-- Make your way to meshery.ui, login with your user details and head over to the local port, found at <code>localhost:3000</code>.
+- Navigate to the Meshery UI, login with your user details and head over to the local port, found at `localhost:9081.`
```

Comment:
The location needs to be an example. localhost:9081 may or may not be where Meshery is deployed.